### DISCUSSION TITLE
Vertical scrollbars at edit components for both text and JS are missing

### DISCUSSION THREAD

**Discussion by gmentzik** [Budibase/budibase#10972] [Help] [answered]
I have checked the last couple of releases and I see that vertical scrollbar is missing from text or Js code fields from most components I checked (eg Paragraph text, Update state button action, Embed etc...). When adding text the window get bigger (larger in height) and you can only scroll text with up/down arrow keys or page up page down. Is this a desired behavior or something was missed after adding new features to the component text/JS editor window?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @gmentzik,

I've replicated the issue you described. I've created an issue for this here; https://github.com/Budibase/budibase/issues/10990. Hopefully it will be swiftly fixed. Thanks for bringing it to our attention!

**Reply by gmentzik** [reply]
@ConorWebb96 
I confirm this is fixed at 2.8.10
